site/student-lectures/Search-Space-Analysis/Search Space Analysis.rst

```diff
@@ -0,0 +1,208 @@
+**********************************************
+Search Space Analysis in Computational Systems
+**********************************************
+
+Introduction to Search Spaces
+----------------------------
+
+Search Space Terminology 
+------------------------
+* search space size #S
+* number of (optimal) solutions #S' ,solution density #S'/#S
+* search space diameter (:math:`G_{N}`) = maximal distance between any two candidate solutions
+* N(:math:`pi`)  is the  neighbourhood relation of a given candaicae slution (:math:`pi`)
+* g(:math:`pi`) is the fitress of a given candaicae slution (:math:`pi`)
```

Comment:
fitress

Reply:
thanks! just installed spell check on vs code to catch everything